client/my-sites/email/email-non-domain-owner-message/index.tsx

```diff
@@ -0,0 +1,139 @@
+		parameters ? stringify( parameters, { addQueryPrefix: true, skipNulls: true } ) : '';
+
+	const buildLoginUrl = () => {
+		const redirectUrlParameter = encodeURIComponent(
+			emailManagementPurchaseNewEmailAccount(
+				selectedSite?.slug ?? '',
+				domain?.name ?? '',
+				null,
+				'login-redirect'
+			)
+		);
+
+		return `/log-in/${ buildQueryString( {
+			email_address: ownerUserName,
+			redirect_to: redirectUrlParameter,
+		} ) }`;
+	};
+
+	const contactOwnerUrl = `https://privatewho.is/${ buildQueryString( {
+		s: domain?.name,
+	} ) }`;
+
+	const loginUrl = buildLoginUrl();
+
+	const onClickLink = ( eventType: 'owner_contact' | 'user_login' | 'contact_support' ) => {
+		const properties = {
+			owner_login: ownerUserName,
+		};
+
+		if ( eventType === 'owner_contact' ) {
+			dispatch( recordTracksEvent( `calypso_email_providers_owner_contact_click`, properties ) );
+
+			return;
+		} else if ( eventType === 'contact_support' ) {
+			dispatch( recordTracksEvent( `calypso_email_providers_contact_support_click`, properties ) );
+
+			return;
+		}
+
+		dispatch( recordTracksEvent( `calypso_email_providers_user_login_click`, properties ) );
+	};
+
+	const translateOptions = {
+		components: {
+			loginLink: (
+				<a
+					rel="noopener noreferrer"
+					href={ loginUrl }
+					onClick={ () => onClickLink( 'user_login' ) }
+				/>
+			),
+			reachOutLink: isPrivacyAvailable ? (
+				<a
+					href={ contactOwnerUrl }
+					onClick={ () => onClickLink( 'owner_contact' ) }
+					rel="noopener noreferrer"
+					target="_blank"
+				/>
+			) : (
+				<></>
+			),
+			contactSupportLink: (
+				<a
+					href="https://wordpress.com/help/contact"
+					onClick={ () => onClickLink( 'contact_support' ) }
+					rel="noopener noreferrer"
+					target="_blank"
+				/>
+			),
+		},
+		args: {
+			ownerUserName,
+			selectedDomainName: domain?.name,
+		},
+	};
+
+	function getReason() {
+		switch ( source ) {
+			case 'email-comparison':
+				return translate(
+					'Email service can only be purchased by {{reachOutLink}}%(ownerUserName)s{{/reachOutLink}}, ' +
+						'who is the owner of %(selectedDomainName)s. ' +
+						'If you have access to that account, please {{loginLink}}log in with the account{{/loginLink}} to make a purchase. ' +
+						'Otherwise, please reach out to {{reachOutLink}}%(ownerUserName)s{{/reachOutLink}} or {{contactSupportLink}}contact support{{/contactSupportLink}}',
+					translateOptions
+				);
+			case 'email-management':
+				return translate(
+					'Additional mailboxes can only be purchased by {{reachOutLink}}%(ownerUserName)s{{/reachOutLink}}, ' +
+						'who is the owner of %(selectedDomainName)s. ' +
+						'If you have access to that account, please {{loginLink}}log in with the account{{/loginLink}} to make a purchase. ' +
+						'Otherwise, please reach out to {{reachOutLink}}%(ownerUserName)s{{/reachOutLink}} or {{contactSupportLink}}contact support{{/contactSupportLink}}',
```

Comment:
```suggestion
						'Otherwise, please reach out to {{reachOutLink}}%(ownerUserName)s{{/reachOutLink}} or {{contactSupportLink}}contact support{{/contactSupportLink}}.',
```
A dot here too.